--- libssh-0.9.6/src/CMakeLists.txt.orig	2021-08-26 14:27:44.000000000 +0200
+++ libssh-0.9.6/src/CMakeLists.txt	2021-09-29 21:52:22.046826365 +0200
@@ -225,7 +225,7 @@
             external/sc25519.c
            )
     endif (NOT HAVE_OPENSSL_ED25519)
-    if(OPENSSL_VERSION VERSION_LESS "1.1.0")
+    if((OPENSSL_VERSION AND OPENSSL_VERSION VERSION_LESS "1.1.0") OR (_OPENSSL_VERSION AND _OPENSSL_VERSION VERSION_LESS "1.1.0"))
         set(libssh_SRCS ${libssh_SRCS} libcrypto-compat.c)
     endif()
 endif (WITH_GCRYPT)
